CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # [Changelog](https://github.com/yola/drf-madprops)
 
+## 0.1.0
```

Comment:
if we're using semantic versioning then this is an api breaking change so you bump the major version